runtime_lib/CMakeLists.txt

```diff
@@ -30,8 +30,6 @@ project(air_runtime_lib LANGUAGES CXX C)
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED YES)
 
-# Eventually we need to have some configuration options to select LibXAIEV2
-set(LibXAIE_ROOT ${Sysroot}/usr ${Sysroot}/opt/xaiengine ${Sysroot}/opt/xaienginev2)
```

Comment:
Where do you think this search path should live?

Reply:
There are some defaults here:
https://github.com/Xilinx/cmakeModules/blob/master/FindLibXAIE.cmake#L36
Or it's given to cmake by the user.